source/gui/settingsDialogs.py

```diff
@@ -2743,6 +2808,23 @@ def _onSoundVolChange(self, event: wx.Event) -> None:
 			wasapi
 			and not self.soundVolFollowCheckBox.IsChecked()
 		)
+		self.soundSplitComboBox.Enable(wasapi)
+		self.soundSplitModesList.Enable(wasapi)
+
+	def isValid(self) -> bool:
+		enabledSoundSplitModes = self.soundSplitModesList.CheckedItems
+		if len(enabledSoundSplitModes) < 1:
+			log.debugWarning("No sound split modes enabled.")

```

Comment:
At least 2 modes are required: there is no point in having a cycling command cycling between only one mode:
```suggestion
		if len(enabledSoundSplitModes) < 2:
			log.debugWarning("Too few sound split modes enabled.")
```

Reply:
But it doesn't hurt at all to have only one mode.